scripts/rules.matterlint

```diff
@@ -95,15 +95,23 @@ load "../src/app/zap-templates/zcl/data-model/chip/resource-monitoring-cluster.x
 load "../src/app/zap-templates/zcl/data-model/chip/sample-mei-cluster.xml";
 
 all endpoints {
-    // These attributes follow a different code path and do not have to be
-    // present in the .matter file
-    //
-    // require global attribute generatedCommandList = 65528;
-    // require global attribute acceptedCommandList = 65529;
-    // require global attribute attributeList = 65531;
+    // These attributes follow separate code paths, but should always be present in matter files:
+    require global attribute generatedCommandList = 65528;
+    require global attribute acceptedCommandList = 65529;
+    require global attribute attributeList = 65531;
 
     require global attribute featureMap = 65532;
     require global attribute clusterRevision = 65533;
+
+    // Deny rules:
+    //   deny cluster Foo;  // denies entire cluster by name
+    //   deny cluster 123;  // denies entire cluster by id
+    //   deny cluster Foo attribute bar;  // denies specific attribute by name
```

Comment:
This syntax mismatches the syntax in the comments above, but the grammar seems to allow it.

Reply:
I updated the syntax last minue, I believe it is ok now.